### PR TITLE
Doug feature volunteer signup

### PR DESCRIPTION
**Pull request by suryasdas**

### CODE REVIEW

File: voluncheer/db.sqlite3
Comment (JinweiZhu) [inline voluncheer/db.sqlite3:1]: Don't push db.sqlite3.

File: voluncheer/profiles/views/organizations.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ def get_context_data(self, **kwargs):
     def form_valid(self, form):
         """Saves the new user and logs them in."""
         user = form.save()
-        user.is_active = False
```
Comment (JinweiZhu) [inline voluncheer/profiles/views/organizations.py:24]: suppose to be false at the beginning. Only true after verification.

File: voluncheer/profiles/views/volunteers.py [hunk 2]
```diff
@@ -23,7 +23,7 @@ def get_context_data(self, **kwargs):
     def form_valid(self, form):
         """Saves the new user and logs them in."""
         user = form.save()
-        user.is_active = False
+        user.is_active = True
```
Comment (JinweiZhu) [inline voluncheer/profiles/views/volunteers.py:26]: suppose to be false at the beginning. Only true after verification.

**Comment by dbleek** [comment]
Not sure if this PR was intentional, this was just a WIP branch for Surya to work from.

**Comment by dbleek** [comment]
Closing since this is not ready for review

**Pull request closed without merging**